builtin/mainmenu/tab_local.lua

```diff
@@ -357,6 +357,40 @@ local function main_button_handler(this, fields, name, tabdata)
 		local game_obj
 		if world then
 			game_obj = pkgmgr.find_by_gameid(world.gameid)
+			if not game_obj then
```

Comment:
The entire body of this `if` should probably be its own function - called something like `try_resolve_alias` maybe?